hack/generator/pkg/astmodel/generation_consts.go

```diff
@@ -8,7 +8,14 @@ package astmodel
 var (
 	CodeGenerationComments []string = []string{
 		// Note that this format is actually an official specification in go:  https://github.com/golang/go/issues/13560
-		"// Code generated by k8s-infra. DO NOT EDIT.", // TODO: Update this when the generated is moved/renamed
+		"// Code generated by azure-service-operator. DO NOT EDIT.", // TODO: Update this when the generated is moved/renamed
+	}
+
+	// Required for backward compatibility.
+	// Presence of any one string will result in the file being treated as a generated file that we are responsible for
+	CodeGenerationCommentsDeprecated []string = []string{
```

Comment:
Is the plan to delete this later?

Feels like we should maybe just delete this now and tell the 3 or 4 of us to manually clean up the directory once when we pull? Cleaner than having legacy here before we're even released?

Reply:
Yeah, didn't take a lot to persuade me. :grin: